Michael-Masas/bootstrap.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+sudo yum install -y yum-utils;
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo;
+sudo yum install -y docker-ce docker-ce-cli containerd.io;
+sudo systemctl start docker;
+sudo systemctl enable docker;
+sudo yum install -y java-1.8.0-openjdk-devel;
+curl --silent --location http://pkg.jenkins-ci.org/redhat-stable/jenkins.repo | sudo tee /etc/yum.repos.d/jenkins.repo;
+sudo rpm --import https://pkg.jenkins.io/redhat-stable/jenkins.io.key;
+sudo yum install -y jenkins;
+sudo systemctl start jenkins;
+sudo systemctl enable jenkins;
```

Comment:
It is more common to enable the service first and than start it.
BTW, Liroy should take a look. I think this issue was not covered by her.

Reply:
i don't see an actual reason  to enable something that isn't running.